open-api/typescript-sdk/src/fetch-client.ts

```diff
@@ -129,7 +129,7 @@ export type AssetResponseDto = {
     /** This property was deprecated in v1.104.0 */
     isReadOnly?: boolean;
     isTrashed: boolean;
-    libraryId: string;
+    libraryId?: string | null;
```

Comment:
Should this be libraryId?, do we want it to be possibly undefined in the response? Surely it should always be string or null, feels like it should remain as libraryId!: but with string | null type

Reply:
This is the auto generated client. It will always match the asset response dto type.

Reply:
Yea... missed that. Should we deprecate libraryId like we have with isReadOnly so we know when we can remove it in future?

Reply:
We can do that yeah.